apps/sso/components/login-card/login-card.tsx

```diff
@@ -105,8 +105,8 @@ export function LoginCard() {
 export default LoginCard;
 
 const StyledLoginCard = styled.div`
-  min-width: 55vw;
-  max-width: 90vw;
+  width: 35vw;
+  height: 73vh;
```

Comment:
Scaling height based on viewport looks weird on other orientations too, I'd just leave it as auto